effdet/loss.py

```diff
@@ -182,12 +219,29 @@ def forward(
             cls_loss = cls_loss.view(bs, height, width, -1, self.num_classes)
             cls_loss *= (cls_targets_at_level != -2).unsqueeze(-1).float()
             cls_losses.append(cls_loss.sum())
+            if not self.use_iou_loss:
+                box_losses.append(_box_loss(
+                    box_outputs[l].permute(0, 2, 3, 1),
+                    box_targets_at_level,
+                    num_positives_sum,
+                    delta=self.delta))
+
+            else:
+                box_outputs_list.append(box_outputs[l].permute(0, 2, 3, 1).reshape([bs, -1, 4]))
+                cls_targets_list.append(cls_targets_at_level.permute(0, 2, 3, 1).reshape([bs, -1, 1]))
+                box_targets_list.append(box_targets_at_level.permute(0, 2, 3, 1).reshape([bs, -1, 4]))
+
+
+        if self.use_iou_loss:
+            # apply bounding box regression to anchors
+            for k in range(box_outputs_list.shape[0]):
```

Comment:
it seems a bug, box_outputs_list is a list, has not shape attribute

Reply:
maybe I did a mistake, `for k in range(len(box_outputs_list)):`